src/durabletask/azext_durabletask/aaz/latest/durabletask/scheduler/_create.py

```diff
@@ -17,8 +17,8 @@
 class Create(AAZCommand):
     """Create a Scheduler
 
-    :example: Create a scheduler in northcentralus
-        az durable-task scheduler create --resource-group testrg --scheduler-name testscheduler --location northcentralus --ip-allowlist "[0.0.0.0/0]" --sku-capacity "1", --sku-name "Dedicated" --tags "{}"
+    :example: Create a scheduler in eastus
+        az durabletask scheduler create --resource-group testrg --scheduler-name testscheduler --location eastus --ip-allowlist "[0.0.0.0/0]" --sku-capacity 1, --sku-name "Dedicated" --tags "{}"
```

Comment:
Need to also remove the comma.
```suggestion
        az durabletask scheduler create --resource-group testrg --scheduler-name testscheduler --location eastus --ip-allowlist "[0.0.0.0/0]" --sku-capacity 1 --sku-name "Dedicated" --tags "{}"
```